cEP-0016.md

```diff
@@ -0,0 +1,445 @@
+fails causing the Travis CI build for corresponding PR to fail as well. Ideally
+the `GitCommitBear` should run on the parent of this merge commit.
+
+Also, there are many special types of commit messages that should be used only
+in conjunction with patches containing a special type of content.
+
+For example, `git revert` commit which is used to revert any previous commit and
+`git merge` commit which is generated whenever someone performs a GitHub rebase
+and tries to merge master onto the pull request. Rebase and merge on GitHub will
+always update the committer information and create new commit SHAs which is not
+correct. These commits generate a default commit message that is very concise
+and not according to coala’s standards. The coala project wants to allow
+`git revert` commits, but does not want to accept `git merge` commits as the
+repositories are strictly fast-forward merges that do not have an extra commit.
+
+In cases where some tests are not required by any pull request made to coala,
+some special sequences can be added to git commit messages to instruct tools to
+operate in a certain way.
+
+This project is about detecting such commits and verifying that they are correct
+according to the configuration settings in `.coafile`
+
+## Background
+
+`.coafile` is a project-wide coala configuration file which is used to specify
+what is checked with which bears for a project.
+
+The workaround that coala uses currently to handle the problem of GitHub Pull
+Request temporary merge commits in `travis.yml` is:
+
+```
+if [[ "$TRAVIS_PULL_REQUEST" != "false" ]]; then
+  sed -e '/bears = GitCommitBear/d' .coafile > .coafile.new
+  mv .coafile.new .coafile
+fi
+```
+
+Below are some code snippets of `.coafile` from different projects depicting
+their use of `GitCommitBear`.
+
+Projects- [coala/coala](https://github.com/coala/coala), [coala/projects](https://github.com/coala/projects), [coala/documentation](https://github.com/coala/documentation)
+
+```
+[commits]
+bears = GitCommitBear
+shortlog_regex = ([^:]*|[^:]+: [A-Z0-9*].*)
+shortlog_trailing_period = False
+```
+
+Projects- [coala/coala-bears](https://github.com/coala/coala-bears), [coala/coala-quickstart](https://github.com/coala/coala-quickstart), [coala/corobo](https://github.com/coala/corobo)
+
+```
+[all.commit]
+bears = GitCommitBear
+shortlog_trailing_period = False
+shortlog_regex = ([^:]*|[^:]+: [A-Z0-9*].*)
+body_close_issue = True
+body_close_issue_full_url = True
+body_close_issue_on_last_line = True
+body_enforce_issue_reference = False
+```
+
+Project- [coala/coala-atom](https://github.com/coala/coala-atom)
+
+```
+[commit]
+bears = GitCommitBear
+```
+
+## Proposed Change
+
+This project would allow organizations to customize the `.coafile`
+to allow following settings in the `[commit]` section.
+
+```
+[commit]
+bears = GitCommitBear, VCSCommitBear, CISkipInspectBear, GitMergeInspectBear, GitRevertInspectBear
```

Comment:
Don't include `VCSCommitBear` here as it's a metabear which means it will be ran implicitly by its dependants